anesthetic/_matplotlib/__init__.py

```diff
@@ -0,0 +1,43 @@
+"""anesthetic override of matplotlib backend for weighted samples."""
```

Comment:
This imports all of the things it needs to from the `matplotlib` backend, and replaces the relevant functions and classes with functions defined in `anesthetic._matplotlib`